### PR TITLE
Also show ko version on presubmit

### PR DESCRIPTION
**Pull request by adrcunha**

### CODE REVIEW

File: scripts/presubmit-tests.sh [hunk 1]
```diff
@@ -296,6 +296,8 @@ function main() {
     kubectl version --client
     echo ">> go version"
     go version
+    echo ">> ko version"
+    ko version
```
Comment (chaodaiG) [inline scripts/presubmit-tests.sh]: this command somehow doesn't work on my machine

Comment (adrcunha) [reply scripts/presubmit-tests.sh]: You must use the latest ko.

**Comment by adrcunha** [comment]
Prow is not using the latest prow-tests image, thus `ko version` doesn't work.

**Comment by adrcunha** [comment]
The version not being displayed is actually caused by an intenal error in ko:

https://github.com/google/ko/blob/master/pkg/commands/version.go#L46

**Comment by adrcunha** [comment]
`ko version` actually displays the hash of its HEAD commit in the local git tree, which is NOT what we want.
We need a better way.
/close

**Pull request closed without merging**